lib/elixir/pages/Syntax Reference.md

```diff
@@ -31,7 +31,26 @@ If the colon is followed by a double- or single-quote, the atom can be made of a
 
 ### Strings
 
-Strings in Elixir are written between double-quotes, such as `"foo"`. Any double-quote inside the string must be escaped with `\ `. Strings support Unicode characters and are stored in UTF-8 encoding.
+Single-line strings in Elixir are written between double-quotes, such as `"foo"`. Any double-quote inside the string must be escaped with `\ `. Strings support Unicode characters and are stored in UTF-8 encoding.
+
+Multi-line strings in Elixir are written with three double-quotes, and can have unescaped quotes within them. The resulting string will end with a newline. The indentation of the line with the initial `"""` on it is used to strip indentation from the inner string. For example:
```

Comment:
Indention is actually based on the last `"""`. :)

Reply:
Fixed